tests/utils/helpers.ts

```diff
@@ -0,0 +1,11 @@
+import path from "path";
+import fs from "fs";
+import { Page } from "@playwright/test";
+
+export async function takeAndSaveScreenshot(fileName: string, page: Page) {
```

Comment:
I don't think there's a need to extract this in the `helpers` file at the moment, we are using this function only once. We can just write this `4` lines of code directly in the test. We can abstract in case we start re-using this logic in multiple places. For now this looks not necessary.